documentation/documentation.html

```diff
@@ -750,14 +886,29 @@ <h2 class="main-h2">Example of Icon with badges</h2>
 
 
 
-            </div>
 
+
+
+            </div>
         </div>
+
+
+
+
+
+
+
+
+
+
```

Comment:
try to remove extra spaces

Reply:
yess that is just for my simplification. will remove in the end